buildSrc/src/main/groovy/Gettext.groovy

```diff
@@ -108,5 +108,27 @@ class Gettext implements Plugin<Project> {
                 }
             }
         }
+
+        def msgattrib_task = project.task('msgattrib') {
+            description = 'The msgattrib program filters the messages of a translation catalog according to their attributes, and manipulates the attributes.'
```

Comment:
Even though this is the description of what msgattrib is doing in general, we're using msgattrib in a very specific way here, so let's change this description to reflect that. Something like `Use msgattrib to remove obsolete strings (that were already removed from the source code & template file) from translation files. ` should be good

Reply:
Updated.